src/tools/miri/cargo-miri/src/phases.rs

```diff
@@ -501,6 +501,15 @@ pub fn phase_runner(mut binary_args: impl Iterator<Item = String>, phase: Runner
     // Set missing env vars. We prefer build-time env vars over run-time ones; see
     // <https://github.com/rust-lang/miri/issues/1661> for the kind of issue that fixes.
     for (name, val) in info.env {
+        // `CARGO_MAKEFLAGS` contains information about how to reach the
+        // jobserver, but by the time the program is being run, that jobserver
+        // no longer exists. Hence we shouldn't forward this.
+        // (This does mean that Miri builds the final crate without a jobserver,
+        // but that is still better than claiming that there is a jobserver at some
+        // file descriptor when there actually isn't.)
```

Comment:
Would following suit?
```
        // FIXME: Miri builds the final crate without a jobserver.
        // This may be fixed with github.com/rust-lang/cargo/issues/12597.
```

Reply:
Yes.